src/test/Pricing.java

```diff
@@ -12,51 +12,52 @@ public class Pricing {
     private static final  String sexSet = sexM;
 
     private static final int ageSet = 13;
-    private static final String eventDaySet = "木曜日";
+    private static final String eventDaySet = "土曜日";
```

Comment:
前指摘忘れていましたが、Setがどういう意図なのか少しわかりかねますが、設定値的な意味なら
```suggestion
    private static final String EVENT_DAY = "土曜日";
```
で伝わるかと思います。
またUPPER_SNAKECASEでの記載をお願いします（finaな定数はすべて横展開お願いします）。

Reply:
今回の場合ConsumerFormの変数の宣言部分はあくまで変数なのでfinalで記述しててもAGEなどの表記はしない。
でしょうか？
むしろprivateでメンバ変数を宣言してるのでfinalをつける意味も無かったでしょうか？理解不足で申し訳ございません。

Reply:
final定数について調べ自分なりに考え修正しました。またご確認よろしくお願いします。